### PR TITLE
TRN-414 complete patch with tests passing.

### PR DESCRIPTION
**Pull request by pchristopher1275**
Ticket found at https://jira2.iparadigms.com/browse/TRN-414
<ticket>
The following list of links/domains each have some kind of endless repeating behavior that makes them useless for us to crawl. Thus these domains are disabled; we only really want to crawl them if we're not going to get junk. These can also be found at https://confluence.iparadigms.com/display/SEU/Walker+Project+Plan
For each one:
Explore whether it's the target site's problem or how we are parsing their links
If we can parse better so that we will no longer have an issue on the site, implement the fix; due to link filtration some of these may no longer be a problem, but at least write a test to verify that
Create a separate story for all "fixed" sites; we'll need to reenable them but in most cases we will probably also want to clear out the bad data too (and remove them from the wiki page)
For sites we cannot fix, update the exclude_reason on those domains to explain why they are excluded
....
</ticket>

### CODE REVIEW

File: walker.yaml [hunk 1]
```diff
@@ -104,6 +104,11 @@ fetcher:
     # this variable is unused.
     http_keep_alive_threshold: 15s
 
+    # The maximum path length that is considered a good url. URL's with paths longer than this will not be crawled. This
+    # variable helps users prevent cycles (where a web-page refers to itself, only with a slightly longer, and hence 
+    # distinct, URI). Set this variable <= 0 to ignore URI path length.
+    max_path_length: 2048
```
Comment (dankinder) [inline walker.yaml]: Can we clarify that "will not be crawled" means we won't insert this link into our database at all? It's a little different, for example, than our behavior with robots.txt-excluded links.

**Comment by pchristopher1275** [comment]
Bump. It looks like we finished up conversation with this on

https://jira2.iparadigms.com/browse/TRN-414

ready to checkin?

File: test/fetcher_test.go [hunk 2]
```diff
@@ -1737,6 +1787,7 @@ func TestParseHttpEquiv(t *testing.T) {
 </head>
 <body>
 Some text here.
+>>>>>>> master
```
Comment (dankinder) [inline test/fetcher_test.go]: Heh, whups. Guess it doesn't cause any issues where it's at now though.

**Comment by dankinder** [comment]
:facepunch:

File: test/fetcher_test.go (hunk 2 above)
Comment (pchristopher1275) [inline test/fetcher_test.go]: Doh!

**Pull request merged**